02.extensions/zooport/02.troubleshooting/02.faq/docs.md

```diff
@@ -0,0 +1,15 @@
+---
+title: FAQ
+taxonomy:
+    category: docs
+---
+
+* **Does the standard ZOO Import/Export support ZOOlanders elements?"** - No, this is currently unsupported. 
+
+* **Is an option "Import from installed software : Joomla / K2" supported?** - CSV import is supported, if the software allows to export the data in CSV. JSON import supports only JSON files generated by ZOOport, due to the specific format. 
```

Comment:
this block is removed here https://github.com/JOOlanders/ZOOport/commit/403d26af7074dc7ab55e259557b1ef9dbd32da80 as is not supported